docker-entrypoint.sh

```diff
@@ -142,22 +142,19 @@ echo "hostname: '${HOSTNAME}'" >> "${raw_cloud_config_path}"
 exec $TASKSET /usr/bin/qemu-system-x86_64 \
   -name ${HOSTNAME} \
   -nographic \
-  -machine accel=kvm -cpu host -smp ${CORES} \
+  -machine accel=kvm \
+  -cpu host,pmu=off \
```

Comment:
adding here pmu=off PMU stands for CPU Performance COunters and amagon cant enable this for us on vmware

Reply:
can you put a comment into the script itself. maybe just link this PR.

Reply:
done